app/models/ext_management_system.rb

```diff
@@ -27,6 +27,9 @@ def self.supported_types_and_descriptions_hash
 
   belongs_to :provider
   belongs_to :tenant
+  include ComplianceMixin
+  include CustomAttributeMixin
+  after_initialize :load_custom_attributes
```

Comment:
@Fryguy the idea is to use lazy loading as @lpichler suggested, just need to make sure it works for me in all the cases.
@blomquisg I think we should enable policies for all kinds of providers, do you see a reason why not to do it?
@lpichler going over that patch, will try it later on today, thanks for the help.